lxd/container_console.go

```diff
@@ -222,9 +223,10 @@ func (s *consoleWs) Do(op *operations.Operation) error {
 
 	// Get the console websocket and close it.
 	s.connsLock.Lock()
-	consolConn := s.conns[0]
+	consoleConn := s.conns[0]
 	s.connsLock.Unlock()
-	consolConn.Close()
+	consoleConn.WriteMessage(websocket.BinaryMessage, []byte("\n\r"))
```

Comment:
@stgraber the `\n\r` sequence is quite unusual, is that on purpose? And why is that different to the `\r` sent further down? Thanks

Reply:
Yeah, it feels a bit odd but just `\n` would insert a new line but still cause the cursor to be way mis-aligned.
`\n\r` causes a line break and resets the cursor to the beginning cleaning things up nicely.